package.json

```diff
@@ -32,7 +32,15 @@
     "split": "scripts/split.js",
     "start": "node -r esm scripts/start/index.js",
     "test": "yarn lint",
-    "version": "conventional-changelog -p angular -i CHANGELOG.md -s && git add CHANGELOG.md"
+    "test:e2e:chromium": "npx rimraf ./reports && testcafe chromium",
+    "test:e2e:chromium:headless": "npx rimraf ./reports && testcafe chromium:headless",
+    "test:e2e:firefox": "npx rimraf ./reports && testcafe firefox",
+    "test:e2e:firefox:headless": "npx rimraf ./reports && testcafe firefox:headless",
+    "version": "conventional-changelog -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
+    "vuepress:build": "yarn run vuepress:prebuild && vuepress build docs",
```

Comment:
side effect @antleblanc ?

Reply:
For sure!
It happened when I was attempting to do a rebase of your branch.

We will remove it right before merging your great work.